<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>credit_calc</class>
 <widget class="QDialog" name="credit_calc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1A1F33;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1" alignment="Qt::AlignRight">
      <widget class="QRadioButton" name="radioButton_annuity">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
	background-color: #353A4E;
	color: white;
	font-family: 'JetBrains Mono';
	padding: 8px 0 8px;
	border-radius: 7px;
	width: 157px;
	max-width: 157px;
}</string>
       </property>
       <property name="text">
        <string>Annuity</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_overpayment">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 1.5px solid #353A4E;
	border-radius: 7px;
	padding: 4px 0 4px;
	margin: 0 1px 0;
	font-family: 'JetBrains Mono';
	color: white;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_6">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: #353A4E;
	color: white;
	font-family: 'JetBrains Mono';
	padding: 8px 0 8px;
	border-radius: 7px;
	min-width: 157px;
}</string>
       </property>
       <property name="text">
        <string>Total payout</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: #353A4E;
	color: white;
	font-family: 'JetBrains Mono';
	padding: 8px 0 8px;
	border-radius: 7px;
	min-width: 157px;
	max-width: 157px;
}</string>
       </property>
       <property name="text">
        <string>Total credit amount</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_total_payout">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 1.5px solid #353A4E;
	border-radius: 7px;
	padding: 4px 0 4px;
	margin: 0 1px 0;
	font-family: 'JetBrains Mono';
	color: white;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: #353A4E;
	color: white;
	font-family: 'JetBrains Mono';
	padding: 8px 0 8px;
	border-radius: 7px;
	min-width: 157px;
}</string>
       </property>
       <property name="text">
        <string>Monthly payment</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1.5px solid #353A4E;
	border-radius: 7px;
	padding: 4px 0 4px;
	margin: 0 1px 0;
	font-family: 'JetBrains Mono';
	color: white;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_monthly_payment">
       <property name="styleSheet">
        <string notr="true">QLabel {
	border: 1.5px solid #353A4E;
	border-radius: 7px;
	padding: 4px 0 4px;
	margin: 0 1px 0;
	font-family: 'JetBrains Mono';
	color: white;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1.5px solid #353A4E;
	border-radius: 7px;
	padding: 4px 0 4px;
	margin: 0 1px 0;
	font-family: 'JetBrains Mono';
	color: white;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: #353A4E;
	color: white;
	font-family: 'JetBrains Mono';
	padding: 8px 0 8px;
	border-radius: 7px;
	min-width: 157px;
}</string>
       </property>
       <property name="text">
        <string>Overpayment</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>157</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>157</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: #353A4E;
	color: white;
	font-family: 'JetBrains Mono';
	padding: 8px 0 8px;
	border-radius: 7px;
	min-width: 157px;
	heigh: 26px;
}</string>
       </property>
       <property name="text">
        <string>Credit term</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="radioButton_differentiated">
       <property name="styleSheet">
        <string notr="true">QRadioButton {
	background-color: #353A4E;
	color: white;
	font-family: 'JetBrains Mono';
	padding: 8px 0 8px;
	border-radius: 7px;
	width: 157px;
	max-width: 157px;
}</string>
       </property>
       <property name="text">
        <string>Differentiated</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">QLabel {
	background-color: #353A4E;
	color: white;
	font-family: 'JetBrains Mono';
	padding: 8px 0 8px;
	border-radius: 7px;
	min-width: 157px;
}</string>
       </property>
       <property name="text">
        <string>Interest rate</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="styleSheet">
        <string notr="true">QLineEdit {
	border: 1.5px solid #353A4E;
	border-radius: 7px;
	padding: 4px 0 4px;
	margin: 0 1px 0;
	font-family: 'JetBrains Mono';
	color: white;
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="pushButton_calculate">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background-color: #353A4E;
	color: white;
	font-family: 'JetBrains Mono';
	padding: 8px 0 8px;
	border-radius: 7px;
	min-width: 157px;
}</string>
       </property>
       <property name="text">
        <string>Ð¡alculate</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>credit_calc</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>credit_calc</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
